star-wars-app/src/People.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default class People extends React.Component {
+  state = { people: [] };
+
+  componentDidMount() {
+    fetch('https://swapi.dev/api/people/')
+    .then(res => res.json())
+    .then(data => this.setState({ people: data.results }));
+  }
+
+  render() {
+    console.log(this.state.people);
+    return (
+      <div>
+        <h1 className="header">Star Wars Characters</h1>
+          <ul>
+            {this.state.people.map((person, id) => (
```

Comment:
This happens to work since the API's IDs are sequential. Another way you can grab the id is to parse the url in each character object. e.g. `person.url.split('/')`, then retrieve the last value in the array.